caffeinate: add -m example (#9187)
tldr-pages/tldr: pages/osx/caffeinate.md

> - Prevent from sleeping until you type Ctrl-C:

Reviewer: ```suggestion
- Prevent from sleeping (use `Ctrl + C` to exit):
```

Reviewer: it's not applied yet

Author: @navarroaxel , It's done now

`caffeinate -i`

> - Prevent disk from sleeping until you type Ctrl-C:

Reviewer: ```suggestion
- Prevent disk from sleeping (use `Ctrl + C` to exit):
```
